pkg/flowaggregator/flowaggregator.go

```diff
@@ -550,3 +565,53 @@ func (fa *flowAggregator) fillK8sMetadata(key ipfixintermediate.FlowKey, record
 		}
 	}
 }
+
+func (fa *flowAggregator) fetchPodLabels(podAddress string) string {
+	pods, err := fa.podInformer.Informer().GetIndexer().ByIndex(podInfoIndex, podAddress)
+	if err == nil && len(pods) > 0 {
```

Comment:
nit: it's recommended to handle error first and make normal path least indent.
```
pods, err := fa.podInformer.Informer().GetIndexer().ByIndex(podInfoIndex, podAddress)
if err != nil {
    klog...
    return ""
}
if len(pods) == 0 {
    return ""
}
pod, ok := pods[0].(*corev1.Pod)
...
labelsJSON, err := json.Marshal(pod.GetLabels())
if err != nil {
    klog.Warningf("JSON encoding of Pod labels failed: %v", err)
    return ""
}
return string(labelsJSON)
```

Reply:
Thanks Quan, addressed.